src/watchers.rst

```diff
@@ -3,24 +3,30 @@ Watchers
 
 Watchers are the parts of ActivityWatch that do all the data collecting.
 
-ActivityWatch comes with two watchers enabled by default:
+
+.. _window-watchers:
+
+Window watchers
+-------------------------------------------
+
+ActivityWatch comes with two watchers enabled by default supporting Windows, macOS and Linux (X11 only):
```

Comment:
Hmm, a bit confusing to list as-watcher-afk under this heading now.

I figured we'd keep the intro and simple nest the Wayland watchers under a Window watchers subheading (underline with `~~~` to create a subheading).